bundle/config/mutator/resourcemutator/apply_target_mode_test.go

```diff
@@ -152,6 +154,13 @@ func mockBundle(mode config.Mode) *bundle.Bundle {
 						},
 					},
 				},
+				SecretScopes: map[string]*resources.SecretScope{
```

Comment:
This test is a good candidate to be converted to acceptance tests (before or after this PR).

Reply:
i suggest to do this in a follow-up PR